Packages/com.unity.inputsystem/InputSystem/Controls/InputControlPath.cs

```diff
@@ -558,15 +558,15 @@ private static bool StringMatches(Substring str, InternedString matchTo)
                         return true; // Wildcard at end of string so rest is matched.
 
                     ++posInStr;
-                    nextChar = char.ToLower(str[posInStr]);
+                    nextChar = char.ToUpperInvariant(str[posInStr]);
 
-                    while (posInMatchTo < matchToLength && matchToLowerCase[posInMatchTo] != nextChar)
+                    while (posInMatchTo < matchToLength && char.ToUpperInvariant(matchToLowerCase[posInMatchTo]) != nextChar)
```

Comment:
This `matchToLowerCase` looks suspicious here.
I think probably it needs renamed to `matchToUpperCase` and have it's contents changed to uppercase?
Then you don't need to uppercase it again here on this line.

Just need to be careful other code is relying on `matchToLowerCase` actually being lower case though.